wiki/osu!tourney/Tournament_Management_Commands/en.md

```diff
@@ -5,6 +5,7 @@ The following chat commands are provided for remote management of multiplayer to
 - `!mp make <name>` - Creates a tournament room with the specified name. A maximum of 4 such rooms may be created.
   - This room is special in that it is not closed when all players have left the room, and it is password protected from players joining this room themselves.
   - When the room is finished, use `!mp close` to close the room.
+- `!mp makeprivate <name>` - Creates a private tournament room with the specified name. This command works the same as `!mp make` but match history is only visible to room's creator and its participants.
```

Comment:
`to room's creator and its participants` --> `to the room's creator and its participants`

i'm not sure whether this was intentionally omitted from the previous review or not but without the `the` the sentence just wouldn't be coherent

Reply:
Ah sorry, I almost forgot. Thank you